prime-router/metadata/hl7_mapping/ORU_R01/base/patient-contact.yml

```diff
@@ -45,13 +45,20 @@ elements:
     schema: datatype/xad-extended-address
     resourceIndex: orderIndex
 
-  - name: phone-number
-    resource: '%resource.telecom'
+  - name: contact-phone-number
+    resource: '%resource.telecom.where(use != "work")'
```

Comment:
Where is use defined and how did you know this was needed?

Reply:
I got the logic from the definition spreadsheet, and `use` is a value in the FHIR resource.